ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/MessageEntity.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MessageEntity extends KsqlEntity {
+
+  private final Optional<String> message;
+
+  public MessageEntity(
+      @JsonProperty("statementText")  final String statementText,
+      @JsonProperty("warnings")       final List<KsqlWarning> warnings,
+      @JsonProperty("message")        final Optional<String> message) {
```

Comment:
Why would the message not be provided?  Isn't the whole point of this entity type to return a message?

```suggestion
      @JsonProperty("message")        final String message) {
```

Reply:
The idea is MessageEntity could be used by a different endpoint which produces an optional message.

E.g. maybe we decide to only print the insert results if the key is null.

Reply:
I don't get it... this is a `MessageEntity`.... what's a `MessageEntity` without a message?